kibana/README.md

```diff
@@ -65,6 +65,7 @@ helm install --name kibana elastic/kibana --version 7.1.0 --set imageTag=7.1.0
 | `nodeSelector`            | Configurable [nodeSelector](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector) so that you can target specific nodes for your Kibana instances                                                                                                                                                                                   | `{}`                                                                                                                      |
 | `tolerations`             | Configurable [tolerations](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/)                                                                                                                                                                                                                                                            | `[]`                                                                                                                      |
 | `ingress`                 | Configurable [ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) to expose the Kibana service. See [`values.yaml`](./values.yaml) for an example                                                                                                                                                                                       | `enabled: false`                                                                                                          |
+| `service`                 | Configurable [service](https://kubernetes.io/docs/concepts/services-networking/service/) to expose the Kibana service. See [`values.yaml`](./values.yaml) for an example                                                                                                                                                                                       | `enabled: false`                                                                                                          |
```

Comment:
I think the `enabled: false` is a copy paste error from the ingress definition above.

```
type: ClusterIP
port: 5601
annotations: {}
```

Should be the defaults

Reply:
I've fixed the defaults on README.md.

Reply:
Also, I've added

- Default empty annotation test case
- Test cases where the value has been added into annotations tag.

Reply:
I'll resolve the comments here as the changes have been made.